crates/aptos/src/node/mod.rs

```diff
@@ -390,3 +408,81 @@ impl CliCommand<Transaction> for LeaveValidatorSet {
             .await
     }
 }
+
+/// Show validator details of the current validator
+#[derive(Parser)]
+pub struct ShowValidatorStake {
+    #[clap(flatten)]
+    pub(crate) profile_options: ProfileOptions,
+    #[clap(flatten)]
+    pub(crate) rest_options: RestOptions,
+    #[clap(flatten)]
+    pub(crate) operator_args: OperatorArgs,
+}
+
+#[async_trait]
+impl CliCommand<serde_json::Value> for ShowValidatorStake {
+    fn command_name(&self) -> &'static str {
+        "ShowValidatorStake"
+    }
+
+    async fn execute(mut self) -> CliTypedResult<serde_json::Value> {
+        let client = self.rest_options.client(&self.profile_options.profile)?;
+        let address = self.operator_args.address(&self.profile_options)?;
+        let response = client
+            .get_resource(address, "0x1::Stake::StakePool")
+            .await?;
+        Ok(response.into_inner())
+    }
+}
+
+/// Show validator details of the current validator
+#[derive(Parser)]
+pub struct ShowValidatorConfig {
+    #[clap(flatten)]
+    pub(crate) profile_options: ProfileOptions,
+    #[clap(flatten)]
+    pub(crate) rest_options: RestOptions,
+    #[clap(flatten)]
+    pub(crate) operator_args: OperatorArgs,
+}
+
+#[async_trait]
+impl CliCommand<serde_json::Value> for ShowValidatorConfig {
+    fn command_name(&self) -> &'static str {
+        "ShowValidatorConfig"
+    }
+
+    async fn execute(mut self) -> CliTypedResult<serde_json::Value> {
+        let client = self.rest_options.client(&self.profile_options.profile)?;
+        let address = self.operator_args.address(&self.profile_options)?;
+        let response = client
+            .get_resource(address, "0x1::Stake::ValidatorConfig")
+            .await?;
+        Ok(response.into_inner())
+    }
+}
+
+/// Show validator details of the validator set
+#[derive(Parser)]
+pub struct ShowValidatorSet {
+    #[clap(flatten)]
+    pub(crate) profile_options: ProfileOptions,
+    #[clap(flatten)]
+    pub(crate) rest_options: RestOptions,
+}
+
+#[async_trait]
+impl CliCommand<serde_json::Value> for ShowValidatorSet {
+    fn command_name(&self) -> &'static str {
+        "ShowValidatorSet"
+    }
+
+    async fn execute(mut self) -> CliTypedResult<serde_json::Value> {
+        let client = self.rest_options.client(&self.profile_options.profile)?;
+        let response = client
+            .get_resource(aptos_root_address(), "0x1::Stake::ValidatorSet")
```

Comment:
Hmmm we'll need to clean this up later as part of removing the root account.

Reply:
Yeah, there's a bunch of stuff we'll need to clean up around that in the future.